pybossa/repositories/task_repository.py

```diff
@@ -434,9 +433,6 @@ def _validate_can_be(self, action, element):
             name = element.__class__.__name__
             msg = '%s cannot be %s by %s' % (name, action, self.__class__.__name__)
             raise WrongObjectError(msg)
-        if isinstance(element, Task) and (action in [self.SAVE_ACTION, self.UPDATE_ACTION]):
```

Comment:
According to the ticket description, we want to add a warning if people do include multiple classifications in import.
maybe we can have:
```
task_levels = (task.info or {}).get('data_access', [])
If task_levels:
    raise Exception / log warnning
```

Reply:
as discussed, added message that will flash message [here](https://github.com/bloomberg/pybossa/pull/484/files#diff-a02a660b7fe836ce948d29476dd500d6R246) on import.